rust/core-lib/src/linewrap.rs

```diff
@@ -432,6 +432,15 @@ impl Lines {
         WrapSummary { start_line, inval_count, new_count, new_soft }
     }
 
+    pub fn logical_line_select_range(&self, text: &Rope, line: usize) -> (usize, usize) {
```

Comment:
I think this could be renamed to something like `logical_line_range` since it might not only be used for selections

Reply:
I have made this change.